packages/api-server/api_server/rmf_io/test_book_keeper.py

```diff
@@ -217,3 +218,67 @@ async def test_write_robot_health(self):
         health = await models.RobotHealth.get(id_="test_robot")
         self.assertIsNotNone(health)
         self.assertEqual(health.health_status, models.HealthStatus.UNHEALTHY)
+
+    async def test_write_task_summary(self):
```

Comment:
Do you know if we now have support to pause/resume tasks as well?

Reply:
I think it is wip.